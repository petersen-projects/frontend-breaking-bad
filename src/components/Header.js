import React from 'react';

const Header = () => (
   <div className="header">
      <div className="header__title">
         Breaking Bad Characters
      </div>
      <div className="header__made-by">
         made by: Oggo Petersen
      </div>
   </div>
);

export default Header;