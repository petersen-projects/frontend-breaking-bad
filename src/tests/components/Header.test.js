import React from 'react';
import { shallow } from 'enzyme';

import Header from '../../components/Header';

test('should render Header correctly', () => {
   const wrapper = shallow(<Header />);
   expect(wrapper).toMatchSnapshot();

   // expect(wrapper.find('div').length).toBe(1);
   // expect(wrapper.find('div').text()).toBe('header');
   // const renderer = new ReactShallowRenderer();
   // renderer.render(<Header />);
   // expect(renderer.getRenderOutput()).toMatchSnapshot();
});
