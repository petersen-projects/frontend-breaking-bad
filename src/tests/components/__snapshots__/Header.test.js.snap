// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="header"
>
  <div
    className="wrapper header__wrapper"
  >
    <div
      className="header__title"
    >
      Breaking Bad Characters
    </div>
    <div
      className="header__made-by"
    >
      made by: Oggo Petersen
    </div>
  </div>
</div>
`;
